### PR TITLE
fix incorrect spelling of "SKIPPED"

### PR DESCRIPTION
**Pull request by DetachHead**
image

### CODE REVIEW

File: src/robot/output/console/dotted.py [hunk 1]
```diff
@@ -79,7 +79,7 @@ def report(self, suite):
                               'test' if not suite.rpa else 'task',
                               plural_or_not(stats.total),
                               secs_to_timestr(suite.elapsedtime/1000.0)))
-        self._stream.highlight(suite.status + 'ED', suite.status)
+        self._stream.highlight(suite.status + ('P' if suite.status == 'SKIP' else '') + 'ED', suite.status)
```
Comment (pekkaklarck) [inline src/robot/output/console/dotted.py:82]: `'PED' if suite.status == 'SKIP' else 'ED'` could be a bit simpler construct here.

**Review by pekkaklarck — COMMENTED** [review]
Good catch! There ought to be tests for this output and they need to be updated as well. I enabled GitHub Actions for this PR so we ought to soon see did anything break.

**Comment by DetachHead** [comment]
it looks like all the actions were cancelled?

**Comment by pekkaklarck** [comment]
Don't know what happened. Possibly GHA didn't like new commits being added when it was running. Activated again.

**Comment by DetachHead** [comment]
https://github.com/robotframework/robotframework/runs/5117910325?check_suite_focus=true

no idea what's going on here

**Comment by pekkaklarck** [comment]
One run failed and I believe others were cancelled after that. There were failures like this:
```
FAIL: Robot.Cli.Console.Console Type.Dotted with width
Lines differ.
Expected: FAILED
Actual:   ED
```
That error seems to be caused by a bug in this PR. I didn't see any test with `SKIPED` failing so it's possible that status isn't tested. Updating tests to cover also that would be good. If it's not easy to see how that can be done, I can do it myself.

**Comment by DetachHead** [comment]
my bad didn't notice that stage failed, i was referring to this:
image

i'll try to fix the other failures

**Comment by pekkaklarck** [comment]
Merged. If you consider this important enough to be added to release notes, please submit an issue as well. I'll then add it to RF 5.0 scope.

**Pull request merged**